### PR TITLE
Invalidation show

### PR DESCRIPTION
**Pull request by aminya**
- [x] Use `ing` for reason
- [x] Use `printstyled` to print the code for methods with color

### CODE REVIEW

File: examples/invalidations_blog.jl [hunk 1]
```diff
@@ -1,5 +1,5 @@
 using SnoopCompile
-using SnoopCompile: countchildren
+import SnoopCompile: countchildren
```
Comment (timholy) [inline examples/invalidations_blog.jl:2]: If I'm *extending* a method I use `import`. If I'm just *calling* a method I use `using`, as a signal that there are no additional method of `countchildren` being defined here. But perhaps I'm missing a deeper reason for this change?

**Comment by timholy** [comment]
Let's do the highlighting later? I'm a little worried about bulking this package up with too many dependencies, especially with https://github.com/JuliaDebug/Cthulhu.jl/pull/82 meaning that it might become a dependency of other packages.

We could also just use plain `printstyled`, no?

**Comment by aminya** [comment]
Yes. Let me apply some quick `printstyled`. The `Highlights` seems like a lot of work!

File: examples/invalidations_blog.jl (hunk 1 above)
Comment (aminya) [reply examples/invalidations_blog.jl:2]: Oh sorry. That is propagated from my closed #90 PR

**Comment by aminya** [comment]
Now it looks easier for me to look at the result 😄

**Pull request merged**